src/loader.c

```diff
@@ -127,6 +127,31 @@ static char * cstr_get_pair(char *p,
 	return cstr_skip_ws(p);
 }
 
+static bool set_auth_dbname(PgDatabase *db, const char *new_auth_dbname)
+{
+	const char *old_auth_dbname = db->auth_dbname;
+
+	if (old_auth_dbname && new_auth_dbname) {
+		if (strcmp(old_auth_dbname, new_auth_dbname) == 0) {
+			return true;
+		}
+
+		free((void *)old_auth_dbname);
```

Comment:
nit: no need for this cast
```suggestion
		free(old_auth_dbname);
```

Reply:
The explicit cast seem to be required here, as otherwise we face with following warning message during compilation:

`warning: passing 'const char *' to parameter of type 'void *' discards qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
                free(old_auth_dbname);`

Reply:
Ah, I didn't realize the const qualifier required it. Maybe add a comment about it.